homestar-invocation/src/authority/issuer.rs

```diff
@@ -91,6 +96,29 @@ where
     }
 }
 
+impl JsonSchema for Issuer {
+    fn schema_name() -> String {
+        "iss".to_owned()
+    }
+
+    fn schema_id() -> Cow<'static, str> {
+        Cow::Borrowed("homestar-invocation::authority::issuer::Issuer")
```

Comment:
also, if this is the pattern, we can const_format (https://docs.rs/const_format/latest/const_format/, which I think we bring in, or could) and use https://doc.rust-lang.org/std/macro.module_path.html and concat it with `Issuer`. This goes for the rest of them @bgins.

Reply:
The creator has an example of this (https://graham.cool/schemars/implementing/):

```
    fn schema_id() -> Cow<'static, str> {
        // Include the module, in case a type with the same name is in another module/crate
        Cow::Borrowed(concat!(module_path!(), "::NonGenericType"))
    }
```

Reply:
Updated to use those 👌